app/views/workflow-ui/job/index.mustache.html

```diff
@@ -70,6 +69,10 @@
                             negate statuses
                         </label>
                     </div>
+                    <div class="input-group mt-2">
+                      <span class="input-group-text">limit</span>
```

Comment:
I think "page limit" might be a bit clearer about what you are limiting here.

Reply:
Actually, the word "limit" kind of implies you won't go past that page, so now I'm confused about the best choice of words here. Maybe "page size" ?

Reply:
Yeah "page size" sounds great to me. I was trying to match the naming of the query parameter but definitely doesn't seem to be the best choice.